extensions/notebook/package.json

```diff
@@ -350,6 +355,7 @@
 		"@types/rimraf": "^2.0.2",
 		"decompress": "^4.2.0",
 		"error-ex": "^1.3.1",
+		"fast-glob": "^3.0.4",
```

Comment:
Please email @yualan so we get this added to ThirdPartyNotices.txt. It's MIT so should be fine.